scripts/require-sync.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
@maxsmythe Do we want to shift towards `Golang` for such new tasks?

Reply:
+1 good point @apeabody that should also let us alleviate the issue with detecting referential constraints.

Reply:
Replaced with a golang version which also has a better check for referential constraints.